go/keybase/main.go

```diff
@@ -61,6 +61,7 @@ func main() {
 		// Had to change from Error to Errorf because of go vet because of:
 		// https://github.com/golang/go/issues/6407
 		g.Log.Errorf("%s", err.Error())
+		os.Exit(1)
```

Comment:
I'll set it to ExitCode_NOTOK if it was previously OK, and then let the following block take care of it.